src/server/server_family.cc

```diff
@@ -264,6 +358,44 @@ void ServerFamily::Load(const std::string& load_path) {
   });
 }
 
+void ServerFamily::SnapshotScheduling(const string &&time) {
+  auto timezone = GetFlag(FLAGS_save_schedule_use_utc) ? absl::UTCTimeZone() : absl::LocalTimeZone();
```

Comment:
lets just use LocalTimeZone for now

Reply:
also CivitTime feels like an over-engineering to me. I do not feel strongly about it but I would just use
```
time_t now = time(NULL);
unsigned hour = (now / 3600) % 24;
unsigned min = (now / 60) % 60;
```
unless I am missing something.

Reply:
Yep that works. I wasn't sure if the expected style was to use abseil or the standard library.